api/v2_search_filters.go

```diff
@@ -49,6 +51,6 @@ type Filter struct {
 }
 
 type TimeFilter struct {
-	StartTime string `json:"startTime,omitempty"`
-	EndTime   string `json:"endTime,omitempty"`
+	StartTime time.Time `json:"startTime,omitempty"`
+	EndTime   time.Time `json:"endTime,omitempty"`
```

Comment:
omitempty doesn't work with time. You'll need to use a pointer here if you'd like to omit the time value